test/e2e/page-objects/pages/test-dapp.ts

```diff
@@ -954,15 +952,6 @@ class TestDapp {
       this.confirmSignatureButtonRedesign,
     );
   }
-
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  async request(method: string, params: any[]) {
```

Comment:
this belongs to the test dapp individual request class, not the test dapp "home" default page